<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Grid</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="gridInputFileLabel">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Choose input file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="gridInputFileLine">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gridBrowseButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="gridOutputFileLabel">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Output file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="gridOutputFileLine">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gridChangeButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Change</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="gridSetParametersLabel">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Set parameters:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="gridParametersTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="accessibleName">
             <string/>
            </property>
            <property name="rowCount">
             <number>1</number>
            </property>
            <property name="columnCount">
             <number>3</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <row/>
            <column/>
            <column/>
            <column/>
            <item row="0" column="0">
             <property name="text">
              <string>Minsup (%)</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>(e.g. 0.25)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gridRunAlgorithmButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Run Algorithm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="gridGraphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>Frequent itemset</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="freqInputFileLabel">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Choose input file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="freqInputFileLine">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="freqBrowseButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="freqOutputFileLabel">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Output file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="freqOutputFileLine">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="freqChangeButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Change</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="freqSetParametersLabel">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Set parameters:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="freqParametersTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="rowCount">
             <number>1</number>
            </property>
            <property name="columnCount">
             <number>3</number>
            </property>
            <row/>
            <column/>
            <column/>
            <column/>
            <item row="0" column="0">
             <property name="text">
              <string>Minsup (%)</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>(e.g. 0.25)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="freqRunAlgorithmButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Run Algorithm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGraphicsView" name="freqGraphicsView"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="freqForwardButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Forward</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>Distance</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1161</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Vector 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leVec1"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbChooseVec1">
             <property name="text">
              <string>Choose</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Vector 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leVec2"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbChooseVec2">
             <property name="text">
              <string>Choose</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Distance method</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Minkowski Distance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Jaccard coefficient</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hamming Distance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cosine Distance</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Parametar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leParam"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pbCompute">
           <property name="text">
            <string>Compute</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Result</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leRes"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab4">
       <attribute name="title">
        <string>Rare itemset</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>1161</width>
          <height>10128</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Choose input file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leApr"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbChooseApr">
             <property name="text">
              <string>Choose</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Choose output file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lePbOutputRare"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbChooseOutput">
             <property name="text">
              <string>Choose</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>MIN SUPPORT</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leMinSupp"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pbFindRare">
           <property name="text">
            <string>Find Rare</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>500</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
